### PR TITLE
Update README.md and CONTRIBUTING.md

### LINKED ISSUE #1035: Update individual Repositories with README.md pointing to new pass documentation site
# What? 

The individual PASS repositories such as `pass-core`, `pass-support` etc., should have a README.md that gives a brief overview of the software and then has a URL to the new PASS documentation site.

# Why?

This will reduce the redundant documentation and standardize where the PASS Documentation is located.

# Acceptance Criteria 

Set of modified/added README.md to the individual PASS repositories.

### PR DESCRIPTION
**Pull request by tsande16**
This PR will cleanup old documentation within the main repository. It updates the top level README with a brief summary and adds URLS pointing to the new documentation site. In addition the CONTRIBUTING.md will be updated to have a brief summary and add a URL pointing to the user docs contributing page. Once merged this will close #1035.

Summary of changes:

- Update top level README
- Update Contributing README
- Remove doc folder which now contains outdated/migrated READMEs and supporting artifacts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,32 @@
...
 
-The PASS platform offers a unified approach for compliance, allowing researchers to fulfil the access policies of their institutions and funders through a single website.
+## Repositories
 
-PASS aims to:
+* [PASS Core](https://github.com/eclipse-pass/pass-core)
+* [PASS UI](https://github.com/eclipse-pass/pass-ui)
+* [PASS Support](https://github.com/eclipse-pass/pass-support)
+* [PASS Acceptance Testing](https://github.com/eclipse-pass)
+* [PASS Docker](https://github.com/eclipse-pass/pass-docker)
```
Comment (rpoet-jh) [inline README.md:28]: Should we include the PASS Documentation repo too?

File: README.md [hunk 2]
```diff
@@ -1,40 +1,32 @@
...
+distinct workflows and submission criteria. PASS simplifies this process by consolidating and streamlining the
+submission workflows, enabling researchers to efficiently meet the diverse compliance requirements.
 
-Meanwhile, many universities have implemented an open access policy, which also requires research outputs to be made publicly accessible. As a result, each publication can be subject to multiple policies each with its own workflow and requirements.
+## Documentation
 
-## Purpose
+* [Eclipse PASS Website](https://eclipse-pass.org/index.html)
+* [PASS Documentation](https://docs.eclipse-pass.org/)
+* [PASS JHU User Guide](https://pass.jhu.edu/guide/)
```
Comment (rpoet-jh) [inline README.md:20]: I'm wondering if we should include a link to the CONTRIBUTING.md file here?

File: README.md [hunk 3]
```diff
@@ -1,40 +1,32 @@
...
 
-The PASS platform offers a unified approach for compliance, allowing researchers to fulfil the access policies of their institutions and funders through a single website.
+## Repositories
 
-PASS aims to:
+* [PASS Core](https://github.com/eclipse-pass/pass-core)
+* [PASS UI](https://github.com/eclipse-pass/pass-ui)
+* [PASS Support](https://github.com/eclipse-pass/pass-support)
+* [PASS Acceptance Testing](https://github.com/eclipse-pass)
```
Comment (rpoet-jh) [inline README.md:27]: Wrong URL

File: README.md (hunk 2 above)
Comment (tsande16) [reply README.md:20]: The contributing file directly links to the contributing page on docs.eclipse-pass.org. Perhaps we just directly link to that and save the user a click?

File: README.md (hunk 1 above)
Comment (tsande16) [reply README.md:29]: Done.

File: README.md (hunk 2 above)
Comment (tsande16) [reply README.md:20]: Done.

File: README.md (hunk 3 above)
Comment (tsande16) [reply README.md]: Done.

**Pull request merged**